SQL/0000-00-00-schema.sql

```diff
@@ -695,8 +695,10 @@ DROP TABLE IF EXISTS `notification_spool`;
 CREATE TABLE `notification_spool` (
   `NotificationID` int(11) NOT NULL auto_increment,
   `NotificationTypeID` int(11) NOT NULL default '0',
-  `TimeSpooled` int(11) NOT NULL default '0',
+  `ProcessID` int(11) NOT NULL DEFAULT '0',
+  `TimeSpooled` datetime DEFAULT NULL,
   `Message` text,
+  `Error` tinyint(1) DEFAULT '0',
```

Comment:
This should be an enum('true', 'false') since QuickForm doesn't work well with tinyint for truthiness comparisons